modules/mongodb/manifests/config.pp

```diff
@@ -4,10 +4,12 @@
 #
 # === Parameters:
 #
+#
```

Comment:
This is probably the most nitpicky comment EVER but do we need the extra hash? We already have bare hash